pages/components/broadcast.html

```diff
@@ -0,0 +1,51 @@
+<html lang="en">
+  <%- include('head.html'); -%>
+  <body>
+    <f-docs-template>
+      <%- include('nav.html'); -%>
+      <main slot="content">
+        <h1>&lt;f-broadcast&gt;</h1>
+
+        <h2>Examples</h2>
+        <p>
+          Broadcast automatically fetches broadcast messages for the current
+          path.
+        </p>
+        <p>Uses f-toast under the hood.</p>
+
+        <h4>Props</h4>
+        <p>
+          Refetch interval: <strong>interval</strong> — default 300 000 (5
+          minutes)
+        </p>
+        <p>Environment: <strong>dev</strong> — default false</p>
+        <p>
+          Url to check for: <strong>path</strong> — default window.location.href
+        </p>
+
+        <syntax-highlight><f-broadcast></f-broadcast></syntax-highlight>
+        <syntax-highlight>
+          // Fetches from dev.finn.no instead of finn.no
+          <f-broadcast dev="true"></f-broadcast
+        ></syntax-highlight>
+        <syntax-highlight>
+          // 4 second refetch interval
+          <f-broadcast interval="4000"></f-broadcast>
+        </syntax-highlight>
+        <syntax-highlight>
+          // The path is automatically set through window.location.href, but you
+          can override this by passing path as a param
+          <f-broadcast path="finn.no/finn/travel"></f-broadcast>
```

Comment:
Should this be just /finn/travel? Path/pathname usually implies the stuff after the origin.

Reply:
I was under the impression, until until I realized [https://dev.finn.no/broadcasts?path=http:/local.finn.no:8080/trygg-handel/bil?finnkode=2](https://dev.finn.no/broadcasts?path=http:/local.finn.no:8080/trygg-handel/bil?finnkode=2) works.

Reply:
https://dev.finn.no/broadcasts suggests `?path=finn.no/finn/travel` too.

Reply:
Rename `path` to `url`, perhaps?

Reply:
Renaming to url seems to be more clear, so can do.

Reply:
Even though the underlying APi says `path` it seems to be some kind of expression: 
![image](https://user-images.githubusercontent.com/440339/136565310-04afcd52-13b2-4e19-8a81-f0359e7092bd.png)
So, what you pass in depends on how the page(s) is defined, really. Might be a path, url, expression. 🤷🏻